### PR TITLE
add alias

### PR DESCRIPTION
**Pull request by hhsnopek**

### CODE REVIEW

**Comment by hhsnopek** [comment]
this is good to go @jescalan

File: lib/index.js [hunk 1]
```diff
@@ -20,7 +20,12 @@ module.exports = function reshapeInclude (options = {}) {
 
       // otherwise, replace the tag with the partial's contents
       const root = options.root || (ctx.filename && path.dirname(ctx.filename)) || ''
-      const includePath = path.join(root, node.attrs.src[0].content)
+      let contentPath = node.attrs.src[0].content
+      if (options.alias) {
+        contentPath = options.alias[contentPath]
+          ? options.alias[contenPath] : contentPath
```
Comment (jescalan) [inline lib/index.js]: typo?

Comment (hhsnopek) [inline lib/index.js]: fixed in 321527c

**Comment by jescalan** [comment]
Awesome, this looks great! Could you just add some docs on it to the readme?

File: README.md [hunk 2]
```diff
@@ -59,6 +59,7 @@ All options are optional, none are required.
 | ---- | ----------- | ------- |
 | **root** | Root path to resolve the include from | the file's path. |
 | **addDependencyTo** | Object with addDependency() method, taking file path as an argument. Called whenever a file is included. | |
+| **alias**| Object with alias mappings, { key: path } | |
```
Comment (jescalan) [inline README.md]: Could I get a slightly more clear description? People will be coming in who have no idea at all what this is 😁

Comment (hhsnopek) [inline README.md]: how's this sound?
> Object with alias mappings, each key is your reference and the corresponding value is the path include's will use to resolve, `{ key: path }`

Comment (jescalan) [inline README.md]: Maybe slightly more explicit with the example. Also how does it resolve the path? Absolute, relative, relative to ?

> Object with alias mappings, each key is your reference and the corresponding value is the path include's will use to resolve, { button: 'node_modules/button-package/button.html' }

Comment (hhsnopek) [inline README.md]: maybe an example is in order here

Comment (hhsnopek) [inline README.md]: > Object with alias mappings, each key is your reference and the corresponding value is the relative path to your file. `{ button: './views/button.html }`

Comment (jescalan) [inline README.md]: Perfecto

**Comment by hhsnopek** [comment]
rebased and ready @jescalan

**Comment by jescalan** [comment]
🎉🎉🎉 thank you!

**Comment by hhsnopek** [comment]
no problem! it was easy like you said 😀

**Pull request merged**